tests/unit/fake_data_root/sos_commands/ceph/ceph_osd_crush_dump

```diff
@@ -207,6 +212,22 @@
             "weight": 6396,
             "alg": "straw2",
             "hash": "rjenkins1",
+            "items": [
+                {
+                    "id": -7,
```

Comment:
same as above and also should this bucket unchanged? the id is changed from 2 -> -7 here.